test/e2e/e2e.go

```diff
@@ -229,6 +230,18 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 
 	f := framework.NewDefaultFramework("efs")
 
+	ginkgo.It("Validating driver installation", func() {
```

Comment:
If there's not a specific reason to have this in a separate Gingko Context than the other tests, than I think we should move this under the existing `ginkgo.Context(storageframework.GetDriverNameWithFeatureTags(driver), func()`

Then, we would have something like 
`ginkgo.It("Validating botocore installation")`

Also, you might want to double check the alignment, I think this block should be indented one more time

Reply:
`ginkgo.It("Validating driver installation")` is kept as generic. So that many packages can be validated in future.